css/style.css

```diff
@@ -51,105 +64,243 @@ ul {
     clip: rect(0 0 0 0);
     overflow: hidden;
 }
+.header {
+    position: relative;
+    max-width: 1160px;
+    height: auto;
+    background-color: #f7e296;
+    margin-top: 40px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 53px;
+}
+.logo-link {
+    position: absolute;
+    left: 60px;
+    top: -21px;
+}
+.header-buttons ul {
+    margin: 0;
+}
 .header-buttons {
     display: flex;
     align-items: center;
     justify-content: space-between;
+    width: auto;
+    min-height: 50px;
+    margin-bottom: 50px;
+    margin-left: 60px;
+    margin-right: 60px;
     font-family: "Opensanslight", sans-serif;
     font-size: 15px;
 }
-.logo-link {
-    margin: 23px 0 0 199px;
-    width: 100%;
-    height: 100%;
+.user-buttons {
+    display: flex;
+    justify-content: space-between;
+    width: 500px;
+}
+.search-container {
+    display: flex;
+    position: relative;
+    box-sizing: border-box;
+    align-items: center;
+    width: 500px;
+}
+.search-container::before {    
+    content: url(.//img/search.svg);
+    position: absolute;
+    top: 17px;
+    z-index: 10;
 }
-.input-search {    
+.input-search {  
+    height: 49px; 
+    padding-left: 36px; 
     opacity: 30%;
     border: none;
+    box-sizing: border-box;
+    background-color: #f7e296;
+}
+.input-search:hover {    
+    opacity: 60%;
+}
+.input-search:focus {
+    opacity: 100%;
+    border-bottom: 2px solid #000000;
+    outline: none;
+
 }
-.search::before {    
-    content: url(.//img/search.svg);    
-    padding: 0 19px 0 0;
+.button-search {
+    display: none;
+}
+.input-search:focus + div {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: 85px;
+    height: 49px;
+    border: 2px solid #000000;
+}
+.user-buttons ul {
+    padding: 0;
 }
 .user-buttons li {
     position: relative;
+    padding-left: 26px;
+}
+.user-buttons li:hover {
+    opacity: 60%;
 }
+/* .user-buttons li:focus {
+    opacity: 30%;
+} */
 .button-registration::before {
     content: url(.//img/user.svg);
     position: absolute;
-    left: -26px;    
+    left: 0px;    
 }
-.button-compare::before {
-    padding: 0 10px 0 0;
-    content: url(.//img/compare.svg);
+.button-compare::before {   
+    content: url(.//img/compare.svg); 
+    position: absolute;  
+    left: 0px;   
 }
 .button-basket::before {
-    padding: 0 11px 0 0;
     content: url(.//img/cart.svg);
+    position: absolute;
+    left: 0px;  
 }
-.user-buttons {
+.user-purchase {
     display: flex;
     justify-content: space-between;
+    width: 220px;
+    padding: 0;
 }
-.user-buttons ul {
+.nav {
     display: flex;
-    justify-content: center;
-    padding: 0 20px;
+    align-items: center;
+    justify-content: space-between;
+    width: auto;
+    padding-bottom: 58px;
+    margin-left: 60px;
+    margin-right: 60px;
+    font-size: 18px;
```

Comment:
bad practice to specify font params here, do it in the selector, related to content of nav or ul, in your case it's probably link

Reply:
What about li that contains link?